wordpress-theme/skyyrose-flagship/assets/css/product-card-v7.css

```
@@ -0,0 +1,139 @@
+/**
+ * V7 lookbook product card — V3 grid display + V6 collection palette
+ * + collection-logo backdrop + auto-advance carousel (hover-pauses in place).
+ * Consumes self-hosted theme fonts; defines its own [data-collection] palette.
+ *
+ * @package SkyyRose
+ */
+
+/* ── collection palette (self-contained; mirrors brand tokens) ── */
+.v7card[data-collection="black-rose"]   { --v7-accent:#C0C0C0; --v7-accent-rgb:192,192,192; --v7-caps:'Cinzel',serif; }
```

Stylelint (17.13.0)

[error] Too many declarations, maximum 1 (declaration-block-single-line-max-declarations)

(declaration-block-single-line-max-declarations)